File: V6.5.py
# @cikey fc86fd6b8ea7c6291f0829f791a2dfa4
# @sid 20251174010007
# @aid V6.5
import random

num = random.randint(1, 100)

palp = int(input("Adivinhe o número: "))

while palp != num:
    if palp < num:
        print("O número é maior")
    else:
        print("O número é menor")
    palp = int(input("Tente novamente: "))

print("Parabéns! Você acertou, o número era: {}".format(num))

File: V65.java
import java.util.Scanner;
import java.util.Random;

public class V65 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        Random rand = new Random();

        int palp;

        int num = rand.nextInt(100) + 1;

        System.out.println("Adivinhe o número: ");
        palp = entrada.nextInt();

        while (palp != num) {
            if (palp > num) {
                System.out.println("O número é menor");
            } else {
                System.out.println("O número é maior");
            }
            System.out.println("Adivinhe o número: ");
            palp = entrada.nextInt();
        }
        System.out.println("Parabéns, você acertou! O número era:" + num);
        entrada.close();
    }
}